### PR TITLE
Module to retrieve data from the divesites API

### PR DESCRIPTION
**Pull request by claravetter**

### CODE REVIEW

**Comment by claravetter** [comment]
done!

**Comment by claravetter** [comment]
did it help?

**Comment by vandenman** [comment]
@claravetter it worked, thanks!

File: diveAPI.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    """ Unit test cases"""
+    def test_location1_cnt1(self):
+        """ Test case: Count dive sites"""
+        _lat = 47.6031537682643
+        _lng = -122.336164712906
+        _dist = 20
+        _sites = dive_sites(_lat, _lng, _dist) 
+        self.assertEqual(len(_sites), 67, "Wrong dive site count")
+
+    def test_location1_cnt2(self):
```
Comment (SvdStappen) [inline diveAPI.py:83]: Seems to me that this test function tests the same thing as the previous one, right? (Only now using different numbers.) I would recommend leaving one test function, and adjusting the values in it if you want to test different value. Same comment applies to the other test functions below.

**Pull request merged**